app/controllers/campaigns_controller.rb

```diff
@@ -22,6 +22,11 @@ def show
   # POST /campaigns
   def create
     @campaign = Campaign.create!(create_params)
+    if params[:has_square_fee]
```

Comment:
Using the Fee id approach I described, this would become
```
if params[:square_fee_id]
  @fee = Fee.where(:id => :square_fee_id)
```

Reply:
I like this too--easier to scale if/when we have different fee structures too :)

Reply:
this makes sense, thank you! adding to the next commit